### PR TITLE
Milestone 2: Events and calculations

### PR DESCRIPTION
**Pull request by eroiyo**
# Added Math Functions

The calculator component was divided in 2 main elements (screen and keyboard) they communicate between the calculator main component, and they are using the Microverse calculation logic files to trade and display information!

## Changes

in the most important changes we have...

- [ ] Now working, you can do math now
- [ ] Logic added
- [ ] a scroll have been added to the screen, in case of big numbers

Thanks for take a look!

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```
Comment (DeliceLydia) [inline src/logic/operate.js:20]: - [ ] When I divide by `zero` let us say `5/0` normally you have to get an `error` in the screen but currently am getting this error
e

Please fix that.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @eroiyo , 

Good job so far! your calculator is working fine, but there is a small issue that you still need to address to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/operate.js (hunk 1 above)
Comment (eroiyo) [reply src/logic/operate.js:20]: Thanks you! i will fix that

**Review by Dipeshtwis — APPROVED** [review]
# Hi @eroiyo, 

### Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Highly recommended
- I would highly recommend adding all the unnecessary files in the `.gitignore` file. Also, remember to avoid pushing those files in Github.
- Add a project description specific to the project to make the README professional.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks, i will apply these changes in the next version, have a nice time

**Pull request merged**